crates/cheatcodes/src/impls/config.rs

```diff
@@ -8,11 +8,19 @@ use foundry_config::{
 };
 use std::path::{Path, PathBuf};
 
+// TODO
+#[derive(Clone, Debug, Default)]
+pub struct EvmOpts {
```

Comment:
hm, this is because we need `Config` and related structs to be on alloy types right? Might tie in with #5986, but this is a cumbersome/significant refactor

Reply:
It's that EvmOpts is in `evm` and we need it before in `cheatcodes`

Reply:
yeah, one reason why we need a few more crates so we can get rid of circular deps